sys/net/network_layer/sixlowpan/lowpan.c

```diff
@@ -1295,9 +1295,6 @@ void lowpan_iphc_decoding(uint8_t *data, uint8_t length, net_if_eui64_t *s_addr,
             memcpy(&(ipv6_buf->srcaddr.uint8[0]), &ll_prefix[0], 2);
             memset(&(ipv6_buf->srcaddr.uint8[2]), 0, 6);
             memcpy(&(ipv6_buf->srcaddr.uint8[8]), &s_addr->uint8[0], 8);
-            /* Invert Universal/local bit as specified in
-             * RFC4291, section 2.5.1 "Interface Identifiers" */
-            ipv6_buf->srcaddr.uint8[8] ^= 0x02;
```

Comment:
I don't have any time to test this right now, but I guess it would work again since it reverts the change in #2497